tests/component/_task_modules/channels/test_ai_channel.py

```diff
@@ -433,6 +618,20 @@ def test___task__add_ai_pos_lvdt_chan___sets_channel_attributes(
     assert chan.ai_ac_excit_freq == voltage_excit_freq
 
 
+def test___task__add_teds_ai_pos_lvdt_chan___sets_channel_attributes(
+    task: Task,
+    sim_position_device: Device,
+    lvdt_teds_file_path,
+):
+    # Our LVDT/RVDT TEDS files are invalid, but we can validate we got deep into the driver.
```

Comment:
I think we should understand what is wrong with these files, or at least create a work item about doing so. The LV TEDS library is able to display these files.

Reply:
I'll debug!, should be fun!

Reply:
Following up - there is a DAQ bug causing this behavior; the files are fine. I'll file that and see if there is another active device that supports these. If not, then this is the best I can do, but I'll fix the comment.